tests/test_sandbox.py

```diff
@@ -0,0 +1,57 @@
+import json
+
+import psycopg2
+import psycopg2.extras
+import pytest
+
+from fixtures import CONFIG, db_cleanup, TEST_DB
+from target_postgres import main
+
+
+def assert_tables_equal(cursor, expected_table_names):
+    cursor.execute("SELECT table_name FROM information_schema.tables WHERE table_schema = 'public'")
+    tables = []
+    for table in cursor.fetchall():
+        tables.append(table[0])
+
+    assert (not tables and not expected_table_names) \
+           or set(tables) == expected_table_names
+
+
+def assert_columns_equal(cursor, table_name, expected_column_tuples):
+    cursor.execute("SELECT column_name, data_type, is_nullable FROM information_schema.columns " + \
+                   "WHERE table_schema = 'public' and table_name = '{}';".format(
+                       table_name))
+    columns = cursor.fetchall()
+
+    assert (not columns and not expected_column_tuples) \
+           or set(columns) == expected_column_tuples
+
+
+class BigCommerceStream:
```

Comment:
Where's the generator for this?

Reply:
See our Slack convo. This just iterates over a real world stream which was snagged for testing etc.

I think this testing file should be reserved for cleansed real world data streams which we are only aiming to replay. The other testing files I think should have generators etc. which try to exercise the other various weirdness.